projects/js-packages/charts/src/components/bar-chart/bar-chart.tsx

```diff
@@ -76,32 +68,34 @@ const BarChart: FC< BarChartProps > = ( {
 	showLegend = false,
 	legendOrientation = 'horizontal',
 	gridVisibility = 'x',
-	renderTooltip = renderDefaultTooltip,
+	renderTooltip,
 	options = {},
 } ) => {
-	const theme = useChartTheme();
-	const { tooltipOpen, tooltipLeft, tooltipTop, tooltipData, hideTooltip, showTooltip } =
-		useTooltip< BarChartTooltipData >();
-	// If we are to spread the options to axis, we need to get rid of tickFormat as it wouldn't work with band scales.
-	const tickFormat = options.axis?.x?.tickFormat ?? formatDateTick;
-
-	const handleMouseMove = useCallback(
-		(
-			event: MouseEvent< SVGRectElement >,
-			value: number,
-			xLabel: string,
-			yLabel: string,
-			seriesIndex: number
-		) => {
-			const coords = localPoint( event );
-			if ( ! coords ) return;
-			showTooltip( {
-				tooltipData: { value, xLabel, yLabel, seriesIndex },
-				tooltipLeft: coords.x,
-				tooltipTop: coords.y - 10,
-			} );
+	const theme = useXYChartTheme( data );
+
+	// Determine the tick format for the x-axis: use user-supplied, or default to label or date formatting.
+	const formatXTick = options.axis?.x?.tickFormat ?? getDefaultXTickFormat( data?.[ 0 ]?.data );
+
+	const renderDefaultTooltip = useCallback(
+		( { tooltipData }: RenderTooltipParams< DataPointDate > ) => {
+			const nearestDatum = tooltipData?.nearestDatum?.datum;
+			if ( ! nearestDatum ) return null;
+
+			return (
+				<div className={ styles[ 'bar-chart__tooltip' ] }>
+					<div className={ styles[ 'bar-chart__tooltip-header' ] }>
+						{ tooltipData?.nearestDatum?.key }
+					</div>
+					<div className={ styles[ 'bar-chart__tooltip-row' ] }>
+						<span className={ styles[ 'bar-chart__tooltip-label' ] }>
+							{ nearestDatum.label || formatXTick( nearestDatum.date.getTime(), 0, [] ) }:
```

Comment:
Moving the `renderDefaultTooltip` function into the BarChart component because we need to format the x value using the formatTick function.